src/cpu/aarch64/jit_generator.hpp

```diff
@@ -190,6 +190,42 @@ class jit_generator : public Xbyak_aarch64::CodeGenerator, public c_compatible {
         L(label);
     }
 
+    template <typename TReg>
+    void uni_fdiv(const TReg &dst, const TReg &src, const TReg &src2) {
+        fdiv(dst, src, src2);
+    }
+
+    void uni_fdiv(const Xbyak_aarch64::VReg4S &dst,
+            const Xbyak_aarch64::VReg4S &src, const Xbyak_aarch64::VReg4S &src2,
+            const Xbyak_aarch64::VReg4S &tmp, const Xbyak_aarch64::PReg &pred) {
+        (void)tmp;
+        (void)pred;
```

Comment:
you can use the `UNUSED` macro from `src/common/z_magic.hpp` for this purpose.

Reply:
Thank you for the suggestion. I'll fix it.